yara-x.yaml

```diff
@@ -1,7 +1,7 @@
 package:
   name: yara-x
-  version: 0.12.0
-  epoch: 5
+  version: "0.13.0"
```

Comment:
Waiting for a green release for `v0.13.0`: https://github.com/VirusTotal/yara-x/actions/workflows/release.yaml

Reply:
We're good: https://github.com/chainguard-dev/malcontent/pull/790